pkg/ccl/backupccl/backup_test.go

```diff
@@ -5790,6 +5790,12 @@ func TestProtectedTimestampsDuringBackup(t *testing.T) {
 	}
 }
 
+func getTableID(db *gosql.DB, dbName, tableName string) (tableID descpb.ID, err error) {
+	err = db.QueryRow(`SELECT table_id FROM crdb_internal.tables WHERE database_name = $1 AND name = $2`,
```

Comment:
Note that we also have `catalogkv.TestingGetTableDescriptor`, which could be useful.

Reply:
good point, changed!